Added setup command (#410)
redwoodjs/redwoodjs.com: docs/cliCommands.md

```

### util

> WARNING: This command is deprecated.  
> Use "yarn rw setup" command. See setup command [here](https://redwoodjs.com/reference/command-line-interface#setup)

Reviewer: ```suggestion
This command is **deprecated** in favor of `yarn rw setup <command>`. See Setup command [here](https://redwoodjs.com/reference/command-line-interface#setup)
```

Let's try that text above (you can commit or change directly). 

Also, let' remove the rest of the information from this section. See screenshot for red crossed out copy to remove:
<img width="687" alt="Screen Shot 2020-10-12 at 4 58 20 PM" src="https://user-images.githubusercontent.com/2951/95800583-90e88d80-0cac-11eb-8952-6e9412a87456.png">

## redwood-tools (alias rwt)

Redwood's companion CLI development tool. You'll be using this if you're contributing to Redwood. See [Contributing](https://github.com/redwoodjs/redwood/blob/main/CONTRIBUTING.md#cli-reference-redwood-tools) in the Redwood repo.  

> ## setup command

Reviewer: ```suggestion
## setup
```

Just setup is fine

> The setup command docs is a comprehensive reference of the Redwood CLI. See the [docs](https://redwoodjs.com/reference/command-line-interface#setup) for refrences.

Reviewer: This content section should look like the current Util command section. So start by removing the content from Util and pasting it here. There are other good examples on the CLI Docs page as well. 

Then change the content to reflect Setup specific information, which comes from the output help of the setup command itself:
```
$ rw setup --help

Initialize project config and install packages

Commands:
  rw setup i18n               Setup i18n
  rw setup tailwind         Setup tailwindcss and PostCSS
  rw setup webpack       Setup webpack in your project so you can add custom config

Options:
  --help     Show help                                                 [boolean]
  --version  Show version number                             [boolean]
```
